### ISSUE TITLE
Could not connect to SharePoint Online: Legacy Auth blocked by ADFS Claim rule

### ISSUE THREAD

**Issue by buidav** [cisagov/ScubaGear#1414]
# 🐛 Summary #

Public Issue #1407 reported that they could not authenticate to SharePoint Online via Interactive Auth and shared this error message 
```
Could not connect to SharePoint Online
```
--------------------------------------------------------------------

Microsoft documentation says this can be caused by a [ADFS claim rule blocking legacy auth](https://learn.microsoft.com/en-us/sharepoint/troubleshoot/administration/connect-sposervice-error)

--------------------------------------------------------------------
Fix is to use Modern Auth by default. Adding these additional parameters to the `Connect-SPOService` call in `Connection.psm1`

```
Connect-SPOService -Url https://REPLACEWITHYOURTENANTNAME-admin.sharepoint.com -ModernAuth $true -AuthenticationUrl https://login.microsoftonline.com/organizations
```

Don't think we need to continue supporting the legacy auth for SPO. 
So, no need for any conditionals or retry mechanism for Legacy auth in case Modern Auth fails. 

## To reproduce ##

Requires ADFS claim rule blocking legacy auth to recreate this error.

## Expected behavior ##

Successful SharePoint Authentication

**Comment by FollyBeachGurl** [comment]
- Added to next week's refinement agenda

**Comment by FollyBeachGurl** [comment]
During refinement discussion 

- discussed potentially adding to troubleshooting 
- don't do anything with active directory 
- low priority 
- documentation
